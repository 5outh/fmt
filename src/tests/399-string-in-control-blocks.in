<?php
//passes: SpaceAroundControlStructures

if (true) {
	$label = "{$foo}{$bar}";
}
