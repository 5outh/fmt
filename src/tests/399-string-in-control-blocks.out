<?php
//passes: SpaceAroundControlStructures

if (true) {
	$label = "{$foo}{$bar}";
}

echo <<<EOT
{$foo}{$bar}
EOT;
